module-2/backend/week_one.md

```diff
@@ -4,31 +4,97 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
+List the five common HTTP verbs and what the purpose is of each verb.
+
+* *get -- render/display on the html page*
+* *post -- get the parameters submitted by a form on the html page and create a new entry in the database*
+* *put -- get the parameters submitted by a form on the html page and update an existing entry in the database*
+* *delete -- html form that when submitted, prompts the database to destroy the selected entry*
+* *patch -- no idea*

```

Comment:
In practice, PATCH is basically synonymous with PUT, though some will tell you that PUT should be used to replace a resource in full, while PATCH should be used to just update some attributes.